components/switchCampuses/index.js

```diff
@@ -35,7 +35,7 @@ export default class SwitchCampuses extends Component {
   }
 
   render() {
-    if (this.props.visiblityState) {
+    if (this.props.visiblityState && this.props.isSwitchAvailableIndestination) {
```

Comment:
this prop var breaks camel case notation ("Indestination")

Reply:
Fixed! Was unrelated to this user story bu I still fixed it :)